src/main/cpp/RobotContainer.cpp

```diff
@@ -16,8 +14,10 @@ void RobotContainer::ConfigureBindings() {
   driverController.A().OnFalse(IndexerStop(&indexer));
   driverController.B().WhileTrue(IndexerEject(&indexer));
   driverController.B().OnFalse(IndexerStop(&indexer));
-}
 
-frc2::CommandPtr RobotContainer::GetAutonomousCommand() {
```

Comment:
Same as above auto command comment (you can leave your code in the Configure Bindings function, the diff display is just weird)